prinz_h2o/src/main/scala/pl/touk/nussknacker/prinz/h2o/repository/H2OModelPayload.scala

```diff
@@ -0,0 +1,14 @@
+package pl.touk.nussknacker.prinz.h2o.repository
+
+import java.io.InputStream
+
+// Tip: MOJO readers also support java.net.URL in place of InputStream
+final case class H2OModelPayload(inputStream: InputStream, name: String, version: String)
+
+object H2OModelPayload {
+
+  def apply(inputStream: InputStream, name: String, version: String): H2OModelPayload = {
```

Comment:
for me there is no sense in such construction - ou use case class or ordinal with its object and apply definition. Whar's the use case of this code? final case class is enough (hink if it should not be closeable like inptstream it contains???)

Reply:
I went with a construct similar to PMML repository. Changed to a minimal class for now.